sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb

```diff
@@ -201,19 +453,388 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Next Steps\n",
-    "You can see further examples of running a pipeline job [here](../)"
+    "Download the output of a pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(pipeline_job.name, download_path=\".\", output_name=\"best_model\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will get the ID of a best run and download the artifacts which goes with it."
```

Comment:
Now we will get the ID of `the` best run and download the artifacts `associated` with it.

Reply:
Fixed.